lib/ConvoyLeader/src/App.cpp

```diff
@@ -180,6 +180,15 @@ void App::setup()
                 {
                     LOG_FATAL("MQTT configuration could not be set.");
                 }
+                else if (0U != settings.getPlatoonVehicleId())

```

Comment:
Please use a static const variable like PLATTOON_LEADER_ID instead of just 0 here.